### PR TITLE
banner crud backend done

### PR DESCRIPTION
**Pull request by harshtatva2002**

### CODE REVIEW

File: CI-API/Controllers/AdminController.cs [hunk 1]
```diff
@@ -325,6 +325,82 @@ public async Task<JsonResult> ApproveRejectDeleteStory([FromBody] AdminPanelStor
...
+
+        #region DeleteBanner
+        /// <summary>
+        /// for deleting the banner data
+        /// </summary>
+        /// <param name="bannerId"></param>
+        /// <returns></returns>
+        [Authorize]
+        [HttpPost("DeleteBanner")]
+        public async Task<JsonResult> DeleteBanner([FromBody] long? bannerId)
```
Comment (UrvishaNayee) [inline CI-API/Controllers/AdminController.cs]: use HttpDelete
change route - DeleteBanner/bannerId and get bannerId from url

File: CI-API/Controllers/AdminController.cs [hunk 2]
```diff
@@ -325,6 +325,82 @@ public async Task<JsonResult> ApproveRejectDeleteStory([FromBody] AdminPanelStor
...
+
+        #region GetBannerDataFromId
+        /// <summary>
+        /// get the banner data from its id
+        /// </summary>
+        /// <param name="bannerId"></param>
+        /// <returns></returns>
+        [Authorize]
+        [HttpGet("GetBannerDataFromId")]
+        public async Task<JsonResult> GetBannerDataFromId(long? bannerId)
```
Comment (UrvishaNayee) [inline CI-API/Controllers/AdminController.cs:372]: update api route to GetBannerDataFromId/bannerId and get bannerId fromUri

**Review by UrvishaNayee — APPROVED** [review]

File: CI-API.Data/Repository/AdminRepository.cs [hunk 3]
```diff
@@ -1106,6 +1106,178 @@ public async Task<JsonResult> DeleteStory(AdminPanelStoryViewModel storyData)
...
 
+        #region banner
+
+        #region GetAllBanners
+        public async Task<JsonResult> GetAllBanners(string? search)
+        {
+            try
+            {
+
+                if (search == null)
```
Comment (vimal-bhalodia-tatvasoft) [inline CI-API.Data/Repository/AdminRepository.cs]: Better to use string.isnullorEmpty and keep habit of when dealing with string

Comment (harshtatva2002) [reply CI-API.Data/Repository/AdminRepository.cs]: changes done sir

**Review by vimal-bhalodia-tatvasoft — APPROVED** [review]

**Pull request merged**